tests/udfs/test_inference.py

```diff
@@ -83,14 +84,15 @@ class TestInferenceUDF(unittest.TestCase):
     def setUp(self) -> None:
         self.udf = InferenceUDF(REDIS_CLIENT)
         self.udf.register_conf("conf1", StreamConf(config_id="conf1"))
+        print(self.udf.get_conf("conf1"))
 
     @patch.object(
         RedisRegistry,
         "load",
         Mock(
             return_value=ArtifactData(
                 artifact=VanillaAE(seq_len=12, n_features=2),
-                extras=dict(version="1", timestamp=time.time(), source="registry"),
+                extras=dict(version="0", timestamp=time.time(), source="registry"),
```

Comment:
Add test cases to assert we are loading correct versions for all the 3 models, also try to add edge cases in this scenario

Reply:
I think testing for all 3 models is out of scope of unit testing.
Let me know what are your thoughts.